app/services/MessageSummaryService.scala

```diff
@@ -95,6 +102,19 @@ class MessageSummaryService {
           None
     }
 
+  private[services] val cancellationDecisionMessage: Reader[Departure, Option[(Message, MessageId)]] =
+    Reader[Departure, Option[(Message, MessageId)]] {
+      departure =>
+        val cancellationDecisionNotifications = getLatestMessageWithoutStatus(departure.messagesWithId)(CancellationDecision)
+
+        val cancellationDecisionNotificationCount = cancellationDecisionNotifications.length
+
+        if (cancellationDecisionNotificationCount > 0 && departureDeclarationCount(departure.messages) == cancellationDecisionNotificationCount)
```

Comment:
I don't think we need the condition to count the departure declarations. Cancellation decisions can be generated by border force and it may be that the user has submitted 2 declarations (first one is rejected) and border force has sent a cancellation decision. In this. case we should still display the cancellation link.